kibana/config/kibana.yml

```diff
@@ -9,5 +9,5 @@ xpack.monitoring.ui.container.elasticsearch.enabled: true
 
 ## X-Pack security credentials
 #
-elasticsearch.username: elastic
-elasticsearch.password: changeme
+xpack.monitoring.elasticsearch.username: elastic
```

Comment:
X-Pack monitoring and X-Pack security are 2 different things.

Reply:
Hmm.. okay If you try to spin up the 7.0.1 stack with the former, I get the error you posted above, is that to do with the `sed` steps in travis?

Reply:
Probably yes, but that's only part of the problem. The options we use right now are perfectly valid, the documentation is clear about it: https://www.elastic.co/guide/en/kibana/current/using-kibana-with-security.html.

Did you maybe want to _add_ those monitoring options to the existing ones?

Reply:
OK, found my answer in https://www.elastic.co/guide/en/kibana/current/monitoring-data.html

> If the Elastic security features are enabled on the monitoring cluster, you must provide a user ID and password so Kibana can retrieve the data. ...

Reply:
I understand what happened now, during my testing I had appended those extra xpack entries instead of replacing them. I had assumed originally that having both sets would conflict, but it seems that is not the case. Updated the PR with the latest commit.